### PR TITLE
Document use of Cloud Native Buildpacks

### PR DESCRIPTION
**Pull request by c0d1ngm0nk3y**
Preview: https://cnb-docs-preview.cfapps.sap.hana.ondemand.com/buildpacks

Note that the content of this page changes!

part-of: https://github.com/cloudfoundry/docs-buildpacks/issues/310

### CODE REVIEW

**Comment by pspinrad** [comment]
Hi @c0d1ngm0nk3y -- good idea to create a sub-level for buildpacks.
In newer Tanzu docs, for example [this page](https://docs.vmware.com/en/VMware-Tanzu-Application-Service/6.0/tas-for-vms/commercial-buildpacks-overview.html), we're using terminology "Cloud Foundry buildpacks" and "Cloud Native buildpacks" to differentiate between the older and newer buildpack types.  Does "Classical" here refer to the CF buildpacks?

**Comment by pspinrad** [comment]
Hi @c0d1ngm0nk3y -- thanks for your continued work on this.  I'm not merging yet here or in docs-book-cloudfoundry because the build pipeline is still broken, sorry for delay!  Once that's fixed, can publish all this great new stuff and verify that it all works.

File: cnb/paketo.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Paketo Buildpacks
+owner: Buildpacks
+---
+
+You can the [paketo](https://paketo.io/) buildpacks as one implementaion of Cloud Native Buildpacks. You will find buildpacks for the most popular languages, like
```
Comment (anthonydahanne) [inline cnb/paketo.html.md.erb:6]: Typo:
```suggestion
You can use the [paketo](https://paketo.io/) buildpacks as one implementation of Cloud Native Buildpacks. You will find Paketo buildpacks for the most popular languages, like
```

File: cnb/understand-buildpacks.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+<p class="note">
+<span class="note__title"><strong>Note</strong></span>
+You must use cf CLI v9 or later.</p>
+
+
+```
+ <code>cf push APP-NAME --lifecycle cnb -b docker://FIRST-BUILDPACK -b docker://SECOND-BUILDPACK</code>
+```
+
+In this case <em>all</em> buildpacks must `detect` and `build` is called in the given order.
```
Comment (anthonydahanne) [inline cnb/understand-buildpacks.html.md.erb:31]:
```suggestion
In this case <em>all</em> buildpacks must run `detect` and `build` in the given order.
```

**Review by anthonydahanne — APPROVED** [review]

**Comment by anthonydahanne** [comment]
@pivotal-david-osullivan apparently I don't have approving rights here; could you please approve?
I've reviewed it, just few typos to fix
Thanks!

**Review by pivotal-david-osullivan — APPROVED** [review]

**Pull request merged**